docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -223,6 +223,7 @@ Query Parameters:
   -   `before`=(`<container id>` or `<container name>`)
   -   `since`=(`<container id>` or `<container name>`)
   -   `volume`=(`<volume name>` or `<mount point destination>`)
+  -   `network`=(`<network name>`)
```

Comment:
@thaJeztah Yeah... while adding details to api document I too realized it is a good addition. But something that stopped me is whether searching on a network by its id is a real use case? Do you think some one would actually keep track of networks by its ids instead of names?